ssc/cmod_pvwattsv8.cpp

```diff
@@ -199,9 +199,9 @@ static var_info _cm_vtab_pvwattsv8[] = {
         { SSC_OUTPUT,       SSC_STRING,      "location",                       "Location ID",                                 "",          "",                                             "Location",      "*",                       "",                          "" },
         { SSC_OUTPUT,       SSC_STRING,      "city",                           "City",                                        "",          "",                                             "Location",      "*",                       "",                          "" },
         { SSC_OUTPUT,       SSC_STRING,      "state",                          "State",                                       "",          "",                                             "Location",      "*",                       "",                          "" },
-        { SSC_OUTPUT,       SSC_NUMBER,      "lat",                            "Latitude",                                    "deg",       "",                                             "Location",      "*",                       "",                          "" },
-        { SSC_OUTPUT,       SSC_NUMBER,      "lon",                            "Longitude",                                   "deg",       "",                                             "Location",      "*",                       "",                          "" },
-        { SSC_OUTPUT,       SSC_NUMBER,      "tz",                             "Time zone",                                   "hr",        "",                                             "Location",      "*",                       "",                          "" },
+        { SSC_OUTPUT,       SSC_NUMBER,      "lat",                            "Latitude",                                    "DD",       "",                                             "Location",      "*",                       "",                          "" },
```

Comment:
What's the reason for specifying "from weather file" in pvsamv1 but not in pvwattsv8 and windpower (referring to output label).

Reply:
pvsamv1 uses "from weather file" for some outputs because of the POA as input solar resource data option. I decided to use it for these variables as well because they are passed through from the weather file and not really simulation outputs. I did not use "from weather file" for pvwattsv8 or windpower because no other variables in those cmods use it.